sockets/unix_socket.go

```diff
@@ -1,5 +1,51 @@
 // +build !windows
 
+/*
+Package sockets is a simple unix domain socket wrapper.
```

Comment:
This example could be defined as an example : see https://blog.golang.org/examples. But can be done in a follow-up :wink:

Reply:
Ok,  Will do it in a follow-up.